changelog.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+0.1.1 (????-??-??)
```

Comment:
Should this just be released as 1.0? It's hard to determine if this is a feature addition or bug fix using sub-1.0.0 version strings.

Reply:
So far all the curveball packages start with a `0.`. I was waiting with 1 everywhere until, idk.. more stuff is done.. like a complete website .. 

But i can do 0.2 for this =)